plugins/auth-auth0/README.md

```diff
@@ -0,0 +1,136 @@
+# @amplication/plugin-auth-auth0
+
+[![NPM Downloads](https://img.shields.io/npm/dt/@amplication/plugin-auth-auth0)](https://www.npmjs.com/package/@amplication/plugin-auth-auth0)
+
+This plugin helps in integrating [Auth0](https://auth0.com/) into your app generated by Amplication and provides the required configuration files.
+
+## Purpose
+
+Provides a way to integrate Auth0 into your app generated by Amplication by adding the required dependencies and configuration files. [**Auth0**](https://auth0.com/) is an authentication and authorization platform that provides the required tools to secure your applications and services.
+
+## Prerequisites
+
+- [Auth0](https://auth0.com/) account
+- [Create an Auth0 application](#create-an-auth0-application) ( Single Page Application )
+- [Create an Auth0 API](#create-an-auth0-api) ( Machine to Machine )
+- [Create an Auth0 action](#create-an-auth0-action) ( Post Login ) to add the user's name, email in access token
+
+## Elaboration
+
+### Create an Auth0 application
+
+- Go to [Auth0](https://auth0.com/) and create an account.
+- Create a new application of type Single Page Application ( Select the React Quickstart )
+- Change the following values to their keys :-
+
+  - Allowed Callback URLs : http://localhost:3001/auth-callback
+  - Allowed Logout URLs : http://localhost:3001/login
+  - Allowed Web Origins : http://localhost:3001
+
+- Go to the settings tab and copy the values of the following :-
+
+  - Domain
+  - Client ID
+
+  ![Auth0 Application Settings](./assets/auth0-admin-settings.png)
+
+### Create an Auth0 API
+
+- Go to the dashboard and create a new API of type Machine to Machine
```

Comment:
Did you mean application?

Reply:
Yeah, it's an API application. I will update the README.